src/index.ts

```diff
@@ -1042,19 +1045,21 @@ export class ValidationService {
     /**
      * Load default validation providers and scans the entire document when ready.
      * @param options.watch If set to true, a MutationObserver will be used to continuously watch for new elements that provide validation directives.
+     * @param options.addNoValidate If set to true (the default), a novalidate attribute will be added to the containing form in validate elemets.
      */
-    bootstrap(options?: { watch?: boolean, root?: HTMLElement }) {
+    bootstrap(options?: { watch?: boolean, root?: HTMLElement, addNoValidate?: boolean }) {
         options = options || {};
```

Comment:
It may be time to make `options` a property of this object so we can access it from other methods without having to pass each option deep into the call stack.

```suggestion
        this.options = options || {};
```

Reply:
> It may be time to make `options` a property of this object...

I started on this a while ago, will open a PR.